components/contact/inquiry.coffee

```diff
@@ -22,7 +22,7 @@ module.exports = class InquiryView extends ContactView
     formTemplate _.extend locals,
       artwork: @artwork
       user: @user
-      contactGallery: false
+      contactGallery: @model.contact_gallery
```

Comment:
Im not sure how we get to this files flow, I had breakpoint inside this file and never got there...